synfig-studio/src/gui/cellrenderer/cellrenderer_value.cpp

```diff
@@ -338,20 +338,20 @@ CellRenderer_ValueBase::render_vfunc(
 		{
 			Distance x( data.get(Real()), Distance::SYSTEM_UNITS);
 			x.convert( App::distance_system, get_canvas()->rend_desc() );
-			property_text() = x.get_string(real_num_decimals).c_str();
+			property_text() = x.get_string(real_num_decimals,true,true,true).c_str();
 		}
 		else
 		{
 			std::string format = strprintf("%%.%df", real_num_decimals);
-			property_text() = strprintf(format.c_str(), data.get(Real()));
+			property_text() = remove_trailing_zeroes(strprintf(format.c_str(), data.get(Real())));
 		}
 	}
 	else
 	if (type == type_time)
 	{
 		property_text() =
 			data.get(Time()).get_string( get_canvas()->rend_desc().get_frame_rate(),
-				                                         App::get_time_format());
```

Comment:
I suggest to revert this change.

Reply:
oh yes, definitely I moved it around by fault.